pacemaker.spec.in

```diff
@@ -1,77 +1,113 @@
-%global gname haclient
+# Define globals to control package behavior (configure these as desired)
+
+## User and group to use for nonprivileged services
 %global uname hacluster
+%global gname haclient
+
+## Where to install Pacemaker documentation
 %global pcmk_docdir %{_docdir}/%{name}
 
-%global specversion 1
+## Upstream pacemaker version, and its package version 
 %global pcmkversion 1.1.15
-# set following to the actual commit or, for final release, concatenate
-# "pcmkversion" macro to "Pacemaker-" (will yield a tag per the convention)
+%global specversion 1
+
+## Upstream commit (or git tag, such as "Pacemaker-${pcmkversion}"
+## for an official release) to use for this package
```

Comment:
(the note about the eyes was my feelign when writing that sed expression in standard proportial font, monospaced font should be default, really)